packages/react-components/src/spacing/spacingTokens.ts

```diff
@@ -0,0 +1,16 @@
+export const SpacingTokens = {
```

Comment:
Alternatively `SPACING_TOKENS`, hard to say what's our approach here, as it seems more of a `enumeration` than a set of data.
```suggestion
export const SpacingToken = {
```